app/main/views/frameworks.py

```diff
@@ -273,6 +273,89 @@ def framework_start_supplier_declaration(framework_slug):
                            framework_close_date=framework_close_date), 200
 
 
+@main.route('/frameworks/<framework_slug>/declaration/fdo', methods=['GET'])
```

Comment:
I don't think we should have this temporary route at all - there's no way this should ever be released with this in. The POST that sets the reuse flag should, for now, redirect into the main supplier declaration signing flow.  From testing perspective it's easy enough to poke the API to make sure the right value has been set.